test/integration/coders.test.ts

```diff
@@ -23,4 +27,13 @@ describe('LeafValueCoder', () => {
       expect(LeafValueCoder.decode(leaf, FIXED_NUMBER_PREFIX)).to.eql(result);
     });
   });
+
+  const bigints: bigint[] = [0n, 1n, maxInt224, minInt224, -1n];
```

Comment:
let's add few cases for "ordinary" numbers

```suggestion
  const bigints: bigint[] = [minInt224, -123456789.987654321, -33.44, -1n, 0n, 1n, 11, 123.321, maxInt224];
```